client/app/user/user.html

```diff
@@ -47,7 +51,7 @@
       </div>
     </div>
     <div class="row image-row">
-        <div ng-repeat="user in users | filter:searchString | startFrom:currentPage*pageSize | limitTo:pageSize">
+        <div ng-repeat="user in users | filter:searchString | startFrom:currentPage*pageSize | limitTo:pageSize |  orderBy: sortOrder">
```

Comment:
Watch out for the ordering here! you are limiting the length of the list and then sorting, sorting should happen first.